packages/@aws-cdk/aws-servicecatalog/package.json

```diff
@@ -97,7 +97,11 @@
   "awslint": {
     "exclude": [
       "resource-attribute:@aws-cdk/aws-servicecatalog.Portfolio.portfolioName",
-      "props-physical-name:@aws-cdk/aws-servicecatalog.PortfolioProps"
+      "resource-attribute:@aws-cdk/aws-servicecatalog.Product.cloudFormationProductProductName",
+      "resource-attribute:@aws-cdk/aws-servicecatalog.Product.cloudFormationProductProvisioningArtifactIds",
+      "resource-attribute:@aws-cdk/aws-servicecatalog.Product.cloudFormationProductProvisioningArtifactNames",
+      "props-physical-name:@aws-cdk/aws-servicecatalog.PortfolioProps",
```

Comment:
Can you keep Portfolio and Product exclusions grouped together?

Maybe add the Product ones before Portfolio, to make the diff look nicer.